waspc/cli/src/Wasp/Cli/Command/Test/Parser.hs

```diff
@@ -0,0 +1,29 @@
+module Wasp.Cli.Command.Test.Parser (test) where
+
+import Options.Applicative
+  ( CommandFields,
+    Mod,
+    Parser,
+  )
+import qualified Options.Applicative as O
+import Wasp.Cli.Command.Call (Call (Test), TestArgs (TestClient, TestServer))
+import Wasp.Cli.Parser.Util (CommandType (CTForwardOptions), mkCommand, mkWrapperCommand)
+
+test :: Mod CommandFields Call
+test = mkCommand "test" parseTest "Executes tests in your project."
+
+parseTest :: Parser Call
+parseTest = Test <$> parseTestArgs
+  where
+    parseTestArgs =
+      O.subparser $
+        mconcat
+          [ mkWrapperCommand "client" CTForwardOptions (TestClient <$> O.many testRestArgs) "Run your app client tests.",
+            mkWrapperCommand "server" CTForwardOptions (TestServer <$> O.many testRestArgs) "Run your app server tests."
```

Comment:
Sure :+1: ! By hiding, do you mean turning it into an internal/hidden command or we should just drop this command off for now if it's actually not being used?

Reply:
That's a good question, I think any of that is ok! I do wonder why @shayneczyzewski put it though, maybe the idea was that it is visible, so people can be like "oh look, server tests are coming also"? But we can remove it for now and possibly revert if @shayneczyzewski advises so.

Reply:
It can be removed if people want to. I mainly added so (a) we had the internal placeholder for `wasp test server` ready and (b) we had some output for the server command that was hopefully friendlier than "command not found" as I figured people may be tempted to try it after running `wasp test client`. But perhaps neither of those warranted it from the start.

Reply:
Ok, @modotte feel free to do here whatever you like the best and then resolve this comment once you are happy with it.